tcl/subcmd.tcl.in

```diff
@@ -2157,6 +2157,13 @@ proc cmdModuleAutoinit {} {
       @setmanpath@}
    }
 
+   # add Modules init directory to PYTHONPATH if enabled
+   @setpythonpath@if {![isEnvVarDefined PYTHONPATH]} {
```

Comment:
that branch of the `if` is not needed (only add path needed, no setenv as add path will do that if variable does not exist yet)

Reply:
Ah. I see. That seems pretty clear now that I think about that more. The similar chunks above which modify PATH and MANPATH aren't perfect examples for modifying PYTHONPATH. I'm not sure I follow the purpose of the `--ignore-refcount` part of modifying PATH. Thank goodnesss PYTHONPATH doesn't have to handle ensuring we have an "empty" `:` like MANPATH.